runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -72,14 +73,15 @@ public JwtGuardHandler(
         this.issuer = options.issuer;
         this.audience = options.audience;
         this.challenge = options.challenge.orElse(null);
+        this.identity = options.identity;
 
         List<JwtKeyConfig> keysConfig = options.keys;
         if ((keysConfig == null || keysConfig.isEmpty()) && options.keysURL.isPresent())
         {
-            JsonbConfig config = new JsonbConfig()
+            JsonbConfig keyConfig = new JsonbConfig()
                     .withAdapters(new JwtKeySetConfigAdapter());
             Jsonb jsonb = JsonbBuilder.newBuilder()
-                    .withConfig(config)
+                    .withConfig(keyConfig)
```

Comment:
Let's inline the `JsonbConfig` into the parameter of the `.withConfig(...)` method.